SPECS/kata-containers-cc/kata-containers-cc.spec

```
@@ -245,55 +80,78 @@ if [ $1 -eq 1 ]; then # Package install
-%{osbuilder}/modules/*
-%{osbuilder}/tools/*
-
-# remove some scripts we don't use
-%exclude %{osbuilder}/tools/osbuilder/rootfs-builder/alpine
-%exclude %{osbuilder}/tools/osbuilder/rootfs-builder/centos
-%exclude %{osbuilder}/tools/osbuilder/rootfs-builder/clearlinux
-%exclude %{osbuilder}/tools/osbuilder/rootfs-builder/debian
-%exclude %{osbuilder}/tools/osbuilder/rootfs-builder/template
-%exclude %{osbuilder}/tools/osbuilder/rootfs-builder/ubuntu
+%dir %{kata_path}
+%dir %{tools_pkg}
+%dir %{tools_pkg}/tools
+%dir %{tools_pkg}/tools/osbuilder
+%{tools_pkg}/tools/osbuilder/Makefile
+
+%dir %{tools_pkg}/src
+%dir %{tools_pkg}/src/kata-opa
+%{tools_pkg}/src/kata-opa/allow-all.rego
+%{tools_pkg}/src/kata-opa/allow-set-policy.rego
+%dir %{tools_pkg}/src/tarfs
+%{tools_pkg}/src/tarfs/Makefile
+%{tools_pkg}/src/tarfs/tarfs.c
+
+%dir %{tools_pkg}/tools/osbuilder/scripts
+%{tools_pkg}/tools/osbuilder/scripts/lib.sh
+
+%dir %{tools_pkg}/tools/osbuilder/rootfs-builder
+%{tools_pkg}/tools/osbuilder/rootfs-builder/rootfs.sh
+%dir %{tools_pkg}/tools/osbuilder/rootfs-builder/cbl-mariner
+%{tools_pkg}/tools/osbuilder/rootfs-builder/cbl-mariner/config.sh
+%{tools_pkg}/tools/osbuilder/rootfs-builder/cbl-mariner/rootfs_lib.sh
+
+%dir %{tools_pkg}/tools/osbuilder/image-builder
+%{tools_pkg}/tools/osbuilder/image-builder/image_builder.sh
+
+%dir %{tools_pkg}/tools/osbuilder/igvm-builder
+%{tools_pkg}/tools/osbuilder/igvm-builder/igvm_builder.sh
+%dir %{tools_pkg}/tools/osbuilder/igvm-builder/azure-linux
+%{tools_pkg}/tools/osbuilder/igvm-builder/azure-linux/config.sh
+%{tools_pkg}/tools/osbuilder/igvm-builder/azure-linux/igvm_lib.sh
+
+%dir %{tools_pkg}/tools/osbuilder/node-builder
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/Makefile
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/clean.sh
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/common.sh
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/uvm_build.sh
+
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/bin
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/bin/kata-agent
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/lib
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/lib/systemd
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/lib/systemd/system
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/lib/systemd/system/kata-containers.target
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/lib/systemd/system/kata-agent.service
 
 %changelog
+* Fri Sep 20 2024 Manuel Huber <mahuber@microsoft.com> - 3.2.0.azl3-1
+- Upgrade to 3.2.0.azl3 release, using build instructions from fork
```

```suggestion
- Upgrade to 3.2.0.azl3 release, refactor build instructions
```
No need to reference the sources as a fork. From the packaging point of view, you are packaging the sources from the microsoft/kata-containers repository